src/background/user-activity.ts

```diff
@@ -17,7 +18,7 @@ export function scheduleAlarms() {
     // To my understanding, if an alarm with an existing name is created, it's
     // not gonna create duplicate alarms. Only one will be active and that's what we need
     browser.alarms.create('lastActiveCheck', {
-      periodInMinutes: 5, // Is this too frequent or too infrequent?
+      periodInMinutes: 1,
```

Comment:
🤔 Why are we changing this?

Reply:
Because we have a 1minute option now. So alarm every 5 mins will not catch this case